Source: Gentoo, written by Gary Stein
Upstream: No
Reason: restores deprecated GifQuantizeBuffer which some packages (notably libgdiplus) still use
--- a/Makefile	2019-03-28 14:57:23.000000000 -0400
+++ b/Makefile	2019-03-31 23:38:20.700603561 -0400
@@ -67,8 +67,8 @@
 
 $(UTILS):: libgif.a libutil.a
 
-libgif.so: $(OBJECTS) $(HEADERS)
-	$(CC) $(CFLAGS) -shared $(LDFLAGS) -Wl,-soname -Wl,libgif.so.$(LIBMAJOR) -o libgif.so $(OBJECTS)
+libgif.so: $(OBJECTS) $(HEADERS) $(UOBJECTS)
+	$(CC) $(CFLAGS) -shared $(LDFLAGS) -Wl,-soname -Wl,libgif.so.$(LIBMAJOR) -o libgif.so $(OBJECTS) $(UOBJECTS)
 
 libgif.a: $(OBJECTS) $(HEADERS)
 	$(AR) rcs libgif.a $(OBJECTS)
